packages/thumbprint-react/package.json

```
@@ -17,8 +17,8 @@
     },
     "peerDependencies": {
         "prop-types": ">= 15.5.7 < 16.0.0-0",
-        "react": ">= 16.3.2 < 17.0.0-0",
-        "react-dom": ">= 16.3.3 < 17.0.0-0"
+        "react": ">= 16.3.2",
+        "react-dom": ">= 16.3.3"
```

It's good to keep this since a future breaking changes in React will eventually be incompatible with our components. For example, React 18 may drop support for `UNSAFE_componentWillUpdate`.

```suggestion
        "react": ">= 16.3.2 < 18.0.0-0",
        "react-dom": ">= 16.3.3 < 18.0.0-0"
```